src/shared/components/index.ts

```diff
@@ -0,0 +1,9 @@
+export * from './button-bar';
+export * from './close-button';
+export * from './drag-drop';
+export * from './dropdown';
+export * from './form-components';
+export * from './formik-fields';
+export * from './loading';
```

Comment:
Your last commit 7057de8 removed the loading folder
```suggestion

```

Reply:
fixed